src/Data/Models/FuelLog.cs

```diff
@@ -0,0 +1,11 @@
+namespace Data
+{
+    public class FuelLog
+    {
+        public int Id;
+        public int Litres;
+        public int Cost;
```

Comment:
Currency?

Reply:
Dont need currency in the data layer as it will all be in euros here